architecture.tex

```diff
@@ -39,7 +39,7 @@ \subsection{Components}\label{sec-components}
 show in Figure~\ref{fig:arch}.
 %The components fall into three categories: device software,
 %infrastructure, and control.
-We first introduce the components running on all devices that
+We first introduce the components used by all devices that
```

Comment:
I kept "running" because this describes software that is physically executed on devices that participate in Seattle. (Devices with Seattle do *use* other components, but those components are contacted via the network, and run on other machines.)

Reply:
Totally valid point.